recipes/comboboxes/combobox_with_popover/combobox_with_popover.test.js

```diff
@@ -117,6 +120,30 @@ describe('DtRecipeComboboxWithPopover Tests', function () {
       it('should render the input', function () { assert.isTrue(wrapper.findComponent(DtInput).exists()); });
     });
 
+    describe('When label is provided', function () {
+      beforeEach(async function () {
+        scopedSlots = { input: '<input v-bind="props.inputProps" />' };
+        _mountWrapper();
+        _setChildWrappers();
+      });
+
+      it('should provide proper props to input element', function () {
+        console.log(wrapper.find('input').attributes());
+        assert.equal(wrapper.find('input').attributes('label'), basePropsData.label);
+        assert.equal(wrapper.find('input').attributes('size'), basePropsData.labelSize);
+        assert.equal(wrapper.find('input').attributes('description'), basePropsData.description);
+      });
+
+      describe('If `labelVisible` prop is `false`', function () {
+        beforeEach(async function () {
+          await wrapper.setProps({ labelVisible: false });
+        });
+        it('label should be empty', function () {
+          assert.equal(wrapper.find('input').attributes('label'), '');
+        });
```

Comment:
Maybe also check that the aria-label is still set even if labelVisible is false since that could be something that's easily missed.

Reply:
I move this test to input component and added aria-label test here